ocean_lib/ocean/mint_fake_ocean.py

```diff
@@ -46,9 +46,13 @@ def mint_fake_OCEAN(config):
         w = Wallet(web3, private_key=key)
 
         if OCEAN_token.token_balance(w.address) < 1000:
-            OCEAN_token.transfer(
-                w.address, amt_distribute_base, from_wallet=deployer_wallet
-            )
+            if (
+                OCEAN_token.allowance(deployer_wallet.address, w.address)
```

Comment:
Having a sufficient allowance only matters when calling `transferFrom` because the `from_wallet` is different from the token `sender`. When calling `transfer`, the `from_wallet` is the `sender` so they are allowed to send as many of their own tokens as they want. 

Thus, I think this allowance check is unnecessary.